packages/env/lib/commands/install-path.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+/**
+ * Internal dependencies
+ */
+const { readConfig } = require( '../config' );
+
+/**
+ * Logs the path to where wp-env files are installed.
+ */
+module.exports = async function installPath() {
+	const { workDirectoryPath } = await readConfig(
```

Comment:
I think we should check if this path exists before outputting it:
```sh
~/source/gutenberg
$ ./packages/env/bin/wp-env install-path
/home/nt/.wp-env/2ccfc3290b190367751be3c7bce4c188

~/source/gutenberg wp-env/install-path
$ cd ..

~/source # wp-env is not used here
$ ./gutenberg/packages/env/bin/wp-env install-path
/home/nt/.wp-env/e7bffa65797a36b9089db69d81c0879d
```
`readConfig` (if I recall) just generate a config based on everything available. But nothing exists until `wp-env start` has been executed. I think it might be helpful to output an error in the case that the work directory path doesn't exist yet, rather than just showing what it _would_ be.